src/processors/default/createERC721NFTsFromTransfersProcessor.ts

```diff
@@ -101,18 +105,23 @@ const processorFunction = (contracts: NftFactory[]) =>
     const transfersForExistingNfts = transfers.filter(transfer => existingNfts.has(transfer.nftId!));
 
     await clients.db.insert(Table.erc721Transfers, transfersForExistingNfts);
-    await clients.db.updateProcessor(NAME, newCursor);
+    await clients.db.updateProcessor(name, newCursor);
   };
+  
+export const createERC721NFTsFromTransfersProcessor: (chainId: ChainId, contracts: NftFactory[]) => Processor = 
+  (chainId, contracts) => {
+    const cursorName = chainId === ChainId.ethereum ? NAME : `${NAME}_${chainId}`;
```

Comment:
cool this is for legacy support ye?

i think we should keep a running list of legacy functionality/state to be fixed/migrated in future?

Initial list that comes to mind is:
 - Ethereum processor name in createERC721NFTsFromTransfersProcessor.ts
 - Ethereum prefixed artist ids
 - Lowercase instead of case-sensitive ethereum addresses
 - Lowercase solana addresses

Reply:
yeah I think some changes will be required on the FE:

- nftFactory and metaFactory now have an id field, and an address field, so to get the address use the address field not the id field
- nft.contractAddress is now the field to get the contract address on an nft
- nft.nftFactoryId is now the nftFactory id, not the contract address

I don't think the FE uses blocks for anything but all those fields were renamed to createdAtBlockNumber

Reply:
cool i don't think the frontend uses any of those fields atm? @emzet93 ?

Reply:
maybe nft.nftFactoryId was used for the etherscan link, so now should need to be contractAddress?